config/default.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     logger,
     uploads: 'uploads',
     enableExperimentalGraphql: true,
+    secret: 'some-secret',
```

Comment:
I think it's best not to put a default secret in the config in order to avoid accidentally deploying to production with a non-secret secret.

Reply:
Agreed, it's a better idea to keep it only in `development.js`.